analyzer/restapi-plugin/src/main/java/eu/fasten/analyzer/restapiplugin/mvn/api/PackageApi.java

```diff
@@ -0,0 +1,49 @@
+package eu.fasten.analyzer.restapiplugin.mvn.api;
+
+import eu.fasten.analyzer.restapiplugin.mvn.RestApplication;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class PackageApi {
+
+    @Autowired
+    PackageApiService service;
+
+    @GetMapping(value = "/{pkg}/versions", produces = MediaType.APPLICATION_JSON_VALUE)
+    ResponseEntity<String> getPackageVersions(@PathVariable("pkg") String package_name,
+                                              @RequestParam(required = false, defaultValue = "0") short offset,
+                                              @RequestParam(required = false, defaultValue = RestApplication.DEFAULT_PAGE_SIZE) short limit) {
+        return service.getPackageVersions(package_name, offset, limit);
+    }
+
+    @GetMapping(value = "/{pkg}/{pkg_ver}", produces = MediaType.APPLICATION_JSON_VALUE)
+    ResponseEntity<String> getPackageVersion(@PathVariable("pkg") String package_name,
+                                             @PathVariable("pkg_ver") String package_version,
+                                             @RequestParam(required = false, defaultValue = "0") short offset,
+                                             @RequestParam(required = false, defaultValue = RestApplication.DEFAULT_PAGE_SIZE) short limit) {
+        return service.getPackageVersion(package_name, package_version, offset, limit);
+    }
```

Comment:
- I believe the response of this endpoint should be flattened. We expect one PackageVersion entity always, but API returns it with a nested array, like `getPackageVersions` does, but it actually may have multiple instances.
- Is there a point in having pagination for such endpoints? I mean, again, we don't expect multiple instances in return, but just one specific?

Reply:
Ok. Wrong PR. I'll move it to #80.